met-web/src/services/modalService/modalSlice.tsx

```diff
@@ -0,0 +1,27 @@
+import { createSlice, PayloadAction } from '@reduxjs/toolkit';
+import { NotificationModalState } from './types';
+
+const initialState: NotificationModalState = {
+    open: false,
+    data: { header: '', subText: [] },
+    type: '',
+};
+
+export const notificationModalSlice = createSlice({
```

Comment:
This is what notification modal dispatch looks like
 dispatch(
                                    openNotificationModal({
                                        open: true,
                                        data: {
                                            header: 'Test',
                                            subText: ['hello'],
                                            handleConfirm: handleStartSurvey,
                                            handleClose: handleClose,
                                        },
                                        type: 'confirm',
                                    }),
                                )
                            }